integration/test/e2e/e2e_test.go

```diff
@@ -45,6 +45,7 @@ func TestE2e(t *testing.T) {
 		t.Fatal(err)
 	}
 
```

Comment:
Since 2.5 doesn't exist, maybe it would be worth adding a test case like `CheckImageDoesntExistOrFail`.

Reply:
Added